static/app/components/pagination/index.tsx

```diff
@@ -1,97 +0,0 @@
-import {Component, ReactElement} from 'react';
```

Comment:
There wasn't really any reason for this to be separate from the Pagination component 🤷